src/rendering/SceneManager.cc

```diff
@@ -1145,6 +1146,55 @@ rendering::LightPtr SceneManager::CreateLight(Entity _id,
   return light;
 }
 
+/////////////////////////////////////////////////
+rendering::VisualPtr SceneManager::CreateInertiaVisual(Entity _id,
+    const math::Inertiald &_inertia, Entity _parentId)
+{
+  if (!this->dataPtr->scene)
+    return rendering::VisualPtr();
+
+  if (this->dataPtr->visuals.find(_id) != this->dataPtr->visuals.end())
+  {
+    ignerr << "Entity with Id: [" << _id << "] already exists in the scene"
+           << std::endl;
+    return rendering::VisualPtr();
```

Comment:
Return the inertia found if exists

Reply:
I think it's just returning an empty visual since the entity id is already being used by another visual, similar to the [`LightVisual`](https://github.com/ignitionrobotics/ign-gazebo/blob/main/src/rendering/SceneManager.cc#L999-L1004)